tests/unit/test_backends.py

```diff
@@ -177,6 +161,53 @@ def getCallSets(self, variantSetId, pageSize=100):
             request, pageSize, self._backend.searchCallSets,
             protocol.GASearchCallSetsResponse, "callSets")
 
+
+class TestSimulatedBackend(TestAbstractBackend):
+
+    def setUp(self):
+        self._backend = backend.SimulatedBackend(
+            numCalls=100, numVariantSets=10)
+        # TODO arbitrary values
+
+    def tearDown(self):
+        pass
+
+
+class TestPaging(TestSimulatedBackend):
```

Comment:
Some docstring comments would be nice. Just a quick sentence explaining what we want to achieve in this test.